### PR TITLE
Support hudi bootstarpping table

### PR DESCRIPTION
**Pull request by willzgw**
## Description

- #17977

## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Support querying bootstrapped Hudi table in the Hudi connector. ({issue}`17977`)
```

### CODE REVIEW

**Review by ebyhr — COMMENTED** [review]
Please add a test.

File: plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java [hunk 1]
```diff
@@ -272,7 +272,8 @@ private void ensurePartitionLoadedCorrectly(String partition)
                         storePartitionView(partitionPathStr, new ArrayList<>());
                     }
                 }
-                catch (IOException e) {
+                catch (Exception e) {
+
```
Comment (ebyhr) [inline plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java:276]: The style is wrong. Also, why do we need to change this exception? 
```
Error:  src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java:[276] (regexp) RegexpMultiline: Line has trailing whitespace
```

Comment (willzgw) [reply plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java:276]: Sorry, I thought wrong.
Will update it and add test.

**Comment by ebyhr** [comment]
Let me close this PR because this PR has been draft status for 10 months. Please feel free to reopen or send a new PR if you continue the work.

**Pull request closed without merging**